features/area-type-parents.public.feature

```diff
@@ -192,3 +192,49 @@ Background:
         """
 
         And the HTTP status code should be "404"
+
+
+    Scenario: Getting parent area count successfully
+
+        Given the following parents areas count response is available from Cantabular:
+        """
+        {
+             "Dimension": {
+                "count": 1,
+                "categories": [
+                    {
+                        "code": "E12000001",
+                        "label": "Hartlepool"
+                    }
+                ],
+                "variable": {
+                    "name":  "LADCD",
+                    "label": "Local Authority code"
+                }
+            }
+        }
+        """
+
+        When I GET "/population-types/Example/area-types/city/parents/LADCD/areas-count?areas=E12000001,E12000002"
+
+        Then I should receive the following JSON response:
+        """
+        1
```

Comment:
this seems strange ? just to get a number as a response without some kind of object wrapping it